src/types.rs

```diff
@@ -204,6 +205,15 @@ pub struct DirectoryLayout {
     pub casing: FilenameCasing,
 }
 
+impl Default for DirectoryLayout {
+    fn default() -> DirectoryLayout {
+        DirectoryLayout {
+            ty: DirectoryLayoutType::Symstore,
```

Comment:
I would propose `DirectoryLayoutType::Native` as default.

Reply:
That uses symstore for windows, right?

Reply:
Yes.

Reply:
Symstore for windows, GNU build-id for linux and LLDB uuid mapped directories for mac.